cmd/data-crawler.go

```diff
@@ -788,7 +788,6 @@ func (i *crawlItem) applyActions(ctx context.Context, o ObjectLayer, meta action
 	if i.debug {
 		logger.LogIf(ctx, err)
 	}
-	applyActionsLogPrefix := color.Green("applyActions:")
 	if i.heal {
```

Comment:
did you think about not attempting a heal when the lifecycle rule says expired?

Reply:
> did you think about not attempting a heal when the lifecycle rule says expired?

yeah but let's do this in another PR. This seems to be complex enough.

Reply:
Okay @vadmeste